src/result_collection.jl

```diff
@@ -88,7 +89,7 @@ function collect_results!(filename, folder;
         allfiles = String[]
         for (root, dirs, files) in walkdir(folder)
             for file in files
-                push!(allfiles, relpath(joinpath(root,file), datadir()))
+                push!(allfiles, joinpath(root,file))
```

Comment:
Shouldn't there be a `isnothing(rpath)` clause here?

Reply:
And if true use `relpath(joinpath(root, file), rpath))` ?

Reply:
as it is written now all the checks for relative/absolute paths happen in the second loop: https://github.com/JuliaDynamics/DrWatson.jl/blob/ed6cee2110686c771d063923e9f093a77b25cd37/src/result_collection.jl#L104-L108

which is why I suggested using something like `maybe_relpath` (and then also `maybe_joinpath`), but I thought having it all in one loop is easier to read...

Reply:
Sorry, you are correct, I have simply not seen the entire code :)